app/assets/javascripts/backbone/views/project_step_modal_view.coffee

```diff
@@ -68,3 +72,10 @@ class MS.Views.ProjectStepModalView extends Backbone.View
   runAndResetDoneCallback: ->
     @done()
     @done = (->) # Reset to empty function.
+
+  showHideStartDate: ->
+    @precedingStepId = @$('#project_step_schedule_parent_id').val()
```

Comment:
Fine to leave the tag ID as schedule_parent_id b/c that is auto built from the field name. Will refactor later.